### PR TITLE
Add warning when uploading public pokemon

### LINKED ISSUE #285: Add warning message if any pending uploads are set to Public
A message in the upload window with something like the following:

>You have selected Public. Public Pokémon can be downloaded and copied by anyone.

### PR DESCRIPTION
**Pull request by YaManicKill**
Fixes #285

d5f1a0f07c11eec6ceb1a094cd08c055

### CODE REVIEW

**Comment by YaManicKill** [comment]
(I decided not to make it red or anything, because we don't want it to be **too** eye-soreish for those that actually want it to, but it does pop up and make it obvious enough, I think)

File: client/add/pokemon.ctrl.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ module.exports = class PokemonAdd {
     this.lines = [];
   }
 
+  hasPublicPokemon () {
+    return Boolean(this.lines.find((line) => line.visibility === 'public'));
```
Comment (not-an-aardvark) [inline client/add/pokemon.ctrl.js]: Nit: instead of `Boolean(this.lines.find(func))`, it's simpler to use `this.lines.some(func)`.

Comment (YaManicKill) [inline client/add/pokemon.ctrl.js]: I knew there was a function, but I couldn't remember what it was...

**Comment by YaManicKill** [comment]
Nitpick fixed.

**Pull request merged**